uxmal/reko
# Make TypedConstantRewiter work with nested structs

- `TypedConstantRewiterTests`: simplify `Given_Global`
Do not subtract `0x100000` from address
-  Create `TypedConstantRewriter` test with pointer to nested struct field
Expected:
```
    &globals->t100100.r0004
```
But was:
```
    &globals->dw100104
```
- Make `TypedConstantRewiter` work with complex global variables(e.g. structs)
- `ComplexExpressionBuilder.FallbackExpression`: avoid exception if data type size is unknown
